products/compute/terraform.yaml

```diff
@@ -296,7 +304,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       backends.group: !ruby/object:Overrides::Terraform::PropertyOverride
         diff_suppress_func: 'compareSelfLinkRelativePaths'
         custom_flatten: templates/terraform/custom_flatten/guard_self_link.go.erb
-      backends.capacityScaler: !ruby/object:Overrides::Terraform::PropertyOverride
+      backends.failover: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
Do we not still want to default `capacityScaler` from the API?

Reply:
I changed this to match the behavior of `BackendService`.